src/Engine/include/OpenLoco/Engine/Input/ShortcutManager.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <cstddef>
+#include <cstdint>
+#include <vector>
+
+namespace OpenLoco::Input
+{
+    enum class Shortcut : uint32_t;
+}
+
+namespace OpenLoco::Input::ShortcutManager
+{
+    using string_id = uint16_t;
```

Comment:
this is defining string_id within the ShortcutManager namespace. I know its perfectly fine but we should stick to defining it  only ever in the OpenLoco namespace.

Reply:
I'll address that, at some point string_id should be defined properly in Engine somewhere.